hyperswitch/hyperWrapper/HyperPaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -210,6 +210,8 @@ extension PaymentSheet {
         /// - Example: ["card", "external_paypal", "klarna"]
         /// - Note: If you omit payment methods from this list, they’ll be automatically ordered by Hyperswitch after the ones you provide. Invalid payment methods are ignored.
         public var paymentMethodOrder: [String]?
+        
```

Comment:
likewise other props please add a short description of this prop

Reply:
done 👍